draft-yusef-tls-pqt-dual-certs.md

```diff
@@ -257,99 +296,46 @@ Policy enforcement regarding the use of dual certificates is implementation-defi
 
 A single composite certificate chain and signature such as defined by {{?TLS-COMPOSITE-MLDSA=I-D.tls-reddy-composite-mldsa}} MAY be an acceptable alternative during post-quantum transition period as long as corresponding signature scheme is listed in `signature_algorithms` extension.
 
+Additional policy examples are given in {{sec-policy-examples}}.
+
 # Performance Considerations
 
-The use of dual certificates increases the size of individual certificates, certificate chains, and associated signatures, which can result in significantly larger TLS handshake messages. These larger payloads may cause packet fragmentation, retransmissions, and handshake delays, especially in constrained or lossy network environments.
+The use of dual certificates increases the size of the certificate and certificate verify messages, which can result in larger TLS handshake messages. These larger payloads may cause packet fragmentation, retransmissions, and handshake delays, especially in constrained or lossy network environments.
```

Comment:
```suggestion
The use of dual certificates increases the size of the Certificate and CertificateVerify messages, which can result in larger TLS handshake messages. These larger payloads may cause packet fragmentation, retransmissions, and handshake delays, especially in constrained or lossy network environments.
```

Reply:
So I left it this way because we actually mean `DualCertificateVerify`, but then the sentence is weird because you can't "increase the size" of something that's new. I know this sentence is a bit awkward, but I don't really have suggestions.